src/index.js

```diff
@@ -3,14 +3,21 @@ import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom'
 import './index.scss';
 import App from './App/App';
+import { ApolloClient, InMemoryCache, ApolloProvider, useLazyQuery } from '@apollo/client';
 // import reportWebVitals from './reportWebVitals';
 
+
+const client = new ApolloClient({
```

Comment:
We will likely do this on a future branch. There is already a branch open where we are making the network requests with GraphQL, so I don't want to do anything with that here. I only changed this part because it broke our page when the ApolloProvider was inside of App.